roles/kubernetes-apps/persistent_volumes/aws/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Kubernetes Persistent Volumes | Lay down AWS gp2 Storage Class template
+  template:
+    src: "aws-storage-class.yml.j2"
+    dest: "{{ kube_config_dir }}/aws-storage-class.yml"
+  register: manifests
+  when:
+    - inventory_hostname == groups['kube-master'][0]
+
+- name: Kubernetes Persistent Volumes | Add AWS gp2 Storage Class
+  kube:
+    name: storage-class
+    kubectl: "{{ bin_dir }}/kubectl"
+    resource: StorageClass
```

Comment:
nit: we always use short form in kube. there's nothing wrong here, but I would write `sc`